src/components/Button/Button.module.scss

```
@@ -0,0 +1,43 @@
+.btn {
+  font-size: 1.6rem;
+  font-weight: 700;
+  padding: 1.4rem 0;
+  border-radius: 0.6rem;
+  font-family: 'Spoqa Han Sans Neo';
```

```suggestion
  @include Spoqa-16-700;
```
폰트는 글로벌에서 선언된 걸 사용해주세요
다른 부분도 수정 바랍니다.